src/test/java/com/celements/cleverreach/util/CssInlinerTest.java

```diff
@@ -28,7 +29,7 @@ public void testInline_null() {
     try {
       cssInliner.inline(null, "");
       fail("Expecting NPE");
-    } catch (NullPointerException npe) {
+    } catch (CssInlineException | NullPointerException npe) {
```

Comment:
What is expected? A CssInlineException or a NPE? The fail-message suggests that an NPE is expected. The Catch-Statement suggests that, either could happen AND is as expected in the given use case. Can you please fix the Fail-Message or the catch statement?